week1/homework/src/server.js

```diff
@@ -1,20 +1,29 @@
+/* eslint-disable indent */
 'use strict';
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
-function createServer(port) {
+const createServer = () => {
   let state = 10;
-
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+  const server = http.createServer((req, res) => {
+    const RenderResponse = (statusCode, write) => {
+      res.statusCode = statusCode;
+      res.setHeader('Content-Type', 'application/json');
```

Comment:
Since the sent data is always JSON, you could move string conversion `SON.stringify` to inside this function.

Reply:
Done ! +1